editor/script_create_dialog.cpp

```diff
@@ -168,9 +161,9 @@ void ScriptCreateDialog::_load_exist() {
 	String path = file_path->get_text();
 	RES p_script = ResourceLoader::load(path, "Script");
 	if (p_script.is_null()) {
-		alert->get_ok()->set_text(TTR("Ugh"));
+		alert->get_ok()->set_text(TTR("OK"));
 		alert->set_text(vformat(TTR("Error loading script from %s"), path));
-		alert->popup_centered_minsize();
+		alert->popup_centered();
```

Comment:
Because I've set default custom size for Popup Alerts, that is not so tiny as when popup_centered_minsize() is used.
It's hard to read text when it is cramped in window it's size, even notice that some Alert poped up :D hehe

It's actually temp adjustment.
Popup Alerts in Godot Editor all over the place are a bit randomly formated, cramped and not standardized.
I plan to create few standardized Editor Widgets. One of theme would be well adjusted Popup for Editor Alerts but before I do this, this must suffice :)